centreon/www/widgets/src/centreon-widget-resourcestable/properties.json

```diff
@@ -0,0 +1,70 @@
+{
+  "title": "Resources table",
+  "description": "Displays data on resource status and events, centralized in a table.",
+  "icon": "<svg id=\"Calque_2\" data-name=\"Calque 2\" xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 60 60\"><defs><style>.cls-1{fill:none;}.cls-1, .cls-2{stroke-width:0px;}.cls-2{fill:#fff;}</style></defs><g id=\"Calque_1-2\" data-name=\"Calque 1\"><g><rect id=\"Rectangle_2058\" data-name=\"Rectangle 2058\" class=\"cls-1\" width=\"60\" height=\"60\" rx=\"4\" ry=\"4\"/><path class=\"cls-2\" d=\"M49.2,6H10.8c-2.65,0-4.8,2.15-4.8,4.8v38.4c0,2.65,2.15,4.8,4.8,4.8h38.4c2.65,0,4.8-2.15,4.8-4.8V10.8c0-2.65-2.15-4.8-4.8-4.8ZM52.08,18.48v16.1h-13.47v-16.1h13.47ZM23.63,34.58v-16.1h12.99v16.1h-12.99ZM36.61,36.58v15.5h-12.99v-15.5h12.99ZM21.63,18.48v16.1H7.92v-16.1h13.71ZM7.92,49.2v-12.62h13.71v15.5h-10.83c-1.59,0-2.88-1.29-2.88-2.88ZM49.2,52.08h-10.59v-15.5h13.47v12.62c0,1.59-1.29,2.88-2.88,2.88Z\"/></g></g></svg>",

```

Comment:
Clean up the svg in order to extract only its content without styles because it's not compatible with dark mode.
See example on other widgets